src/Microsoft.NetCore.Analyzers/Core/Runtime/DisposableFieldsShouldBeDisposed.cs

```diff
@@ -194,6 +207,17 @@ void addOrUpdateFieldDisposedValue(IFieldSymbol field, bool disposed)
                         }
                     }
                 });
+
+                return;
+
+                // Local functions
+                bool ShouldAnalyze(INamedTypeSymbol namedType)
```

Comment:
Since C# 8 is allowed, maybe this should be a static local function?

Reply:
Yes, hoping to do it across the repo with a separate PR